### PR TITLE
Fill project id in stackdriver adapter config if it is empty

### PR DESCRIPTION
**Pull request by bianpengyuan**
Also update out of date attributes in stackdriver test config yaml.

### CODE REVIEW

File: mixer/adapter/stackdriver/stackdriver.go [hunk 1]
```diff
@@ -23,7 +23,10 @@ import (
 
 	multierror "github.com/hashicorp/go-multierror"
 
+	md "cloud.google.com/go/compute/metadata"
```
Comment (douglas-reid) [inline mixer/adapter/stackdriver/stackdriver.go]: nit: run `bin/fmt.sh` on this file to clean up the imports.

Comment (bianpengyuan) [reply mixer/adapter/stackdriver/stackdriver.go]: Done.

**Pull request merged**